### ISSUE TITLE
Some errors

### ISSUE THREAD

**Issue by ZhangMiaoMew** [theSha1do1w/pseudo-3d-pytorch#1]
Hi, thank you for your work.  I found some syntax errors in your code , like in Data_Loader.py:

def preprocess(self):
        if not os.path.exists(self.output_dir):
            os.mkdir(self.output_dir)
            os.mkdir(self.output_dir, 'train')
            os.mkdir(self.output_dir, 'val')
            os.mkdir(self.output_dir, 'test')

I'm using python3.6,  is there any usages like  os.mkdir(self.output_dir, 'train')?  I am new to python, when I google, the web says that os.mkdir( ) don't have this usage.
 If I was wrong, please tell me.

**Comment by theSha1do1w** [comment]
'os.mkdir(self.output_dir, 'train') ' means that make a directory in 'output_dir/train' path

**Comment by ZhangMiaoMew** [comment]
Thanks for your reply. The paper says P3D models can pretrain on 2D image datasets to resolve limited video dataset problem, do you know how to pretrain the 2D convolution kernels on 2D  image datasets? Does P3D use the same network, train on image datasets or change the network architecture? Anyway, thank you in advance.

**Comment by theSha1do1w** [comment]
in the paper that says 
> 
Take awidely adopted 11-layer 3D CNN, i.e., C3D [31] networks,as  an  example,  the  model  size  reaches  321MB  which  iseven larger than that (235MB) of a 152-layer 2D ResNet(ResNet-152)  [7],  making  it  extremely  difficult  to  train  avery deep 3D CNN. More importantly, directly fine-tuningResNet-152  with  frames  in  Sports-1M  dataset  [10]  mayachieve better accuracy than C3D trained on videos fromscratch as shown in Figure 1. 
>
